test/caption-stream-test.js

```diff
@@ -0,0 +1,283 @@
+(function(window, muxjs, undefined) {
+  'use strict';
+
+  var captionStream;
+
+  module('Caption Stream', {
+    beforeEach: function() {
+      captionStream = new muxjs.mp2t.CaptionStream();
+    }
+  });
+
+  QUnit.skip('parses SEIs larger than 255 bytes', function() {
```

Comment:
The other ones are `not implemented` but this seems like a complete test. Just making sure that you didn't leave the skip in there by accident.